### PR TITLE
Add schema reloading

### PR DESCRIPTION
**Pull request by nechaido**

### CODE REVIEW

File: applications/example/schemas/version.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  version: 1,
+}
```
Comment (tshemsedinov) [inline applications/example/schemas/version.js]: It's ok for now but we need schema name, author, license, checksum in `version.js` and I'd rename it to `.schema` or something like `schema.pkg` @nechaido

**Comment by tshemsedinov** [comment]
Need to rebase @nechaido

File: applications/example/schemas/version.js (hunk 1 above)
Comment (lundibundi) [reply applications/example/schemas/version.js]: Perhaps ms.js?

**Comment by tshemsedinov** [comment]
Landed in 936b616

**Pull request closed without merging**